### ISSUE TITLE
Add some sort of tool tip that describes the Clinical Trial ID is pulled in from PubMed?

### ISSUE THREAD

**Issue by kkrysiak** [genome/civic-client#1333]
This has caused some confusion over time. See comment: https://civicdb.org/events/genes/5177/summary/variants/2958/talk/comments 

Perhaps we can add clarity for now by adding a tool tip since this other issue is farther down the road.
https://github.com/griffithlab/civic-client/issues/1055

**Comment by jmcmichael** [comment]
Yeah, I have an idea for how to clearly indicate that the trial id is associated w/ the pubmed entity, not the evidence entity.
